test/unit/org/apache/cassandra/config/YamlConfigurationLoaderTest.java

```diff
@@ -40,41 +40,43 @@ public void trackWarningsFromConfig()
     {
         // this test just makes sure snakeyaml loads the test config properly and populates the fields (track warnings uses final in some places)
         // if the config is changed, its ok to update this test to reflect that change
-        TrackWarnings tw = load("test/conf/cassandra.yaml").track_warnings;
-        assertThat(tw.enabled).isTrue();
+        Config c = load("test/conf/cassandra.yaml");
```

Comment:
The methods `trackWarningsFromConfig` and `trackWarningsFromMap` could be named `readThresholdsFromConfig` and `readThresholdsFromMap`.